video_management/gopro_concat.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+"""
+GoPro Video Concatenation Script
+
+This script finds and concatenates multi-part GoPro video files stored in an S3 bucket.
+It identifies groups of videos belonging to the same DropID and combines them into single MP4 files.
+
+Usage:
+    python gopro_concat.py --prefix "media/SURVEY_ID" [options]
+    
+    Or make it executable:
+    chmod +x gopro_concat.py
+    ./gopro_concat.py --prefix "media/SURVEY_ID" [options]
+"""
+
+import sys
+import os
+import subprocess
+import argparse
+import logging
+from pathlib import Path
+
+# Configure logging
```

Comment:
No need to configure logging, If you import logging that's enough, you can just use logging.info() and it works. (it's configured in sftk/log_config.py)